src/util.ts

```diff
@@ -24,6 +27,8 @@ if (!disabledFeatures.s3) {
 
 const gRPCAccountChannel = createChannel(`${config.grpc.account.address}:${config.grpc.account.port}`);
 const gRPCAccountClient: AccountClient = createClient(AccountDefinition, gRPCAccountChannel);
+const gRPCFriendsChannel = createChannel(`${config.grpc.friends.address}:${config.grpc.friends.port}`);
```

Comment:
Just to make it more clear at a glance that these are different connections/clients 
```suggestion
const gRPCAccountClient: AccountClient = createClient(AccountDefinition, gRPCAccountChannel);

const gRPCFriendsChannel = createChannel(`${config.grpc.friends.address}:${config.grpc.friends.port}`);
```